app/views/supervision/another_body.scala.html

```diff
@@ -36,39 +35,32 @@
 
     @form(f, controllers.supervision.routes.AnotherBodyController.post(edit)) {
 
-        @fieldset(
-            legend = "supervision.another_body.title",
+        @fieldsetWithErrorGroup(
             f = f("anotherBody"),
-            legendHidden = true,
-            panel = false
+            panel = false,
+            heading="supervision.another_body.title",
+            section="summary.supervision",
+            classes = Seq("inline")
         ) {
             @radio(
                 f = f("anotherBody"),
                 labelText = "lbl.yes",
                 value = "true",
-                target = s"#$target",
-                inline = true
+                target = s"#$target"
             )
             @panel(
                 field = Some(f(s"$target"))
             ) {
-
-                @errorGroup(
-                    field = f("supervisor-panel"),
```

Comment:
Im a little worried about the name of this removed errorGroup field. Do errors still work as expected on this page? If not, we'll take a look :)

Reply:
Seems to work fine.